geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ServerConnection.java

```diff
@@ -1205,6 +1206,8 @@ public void run() {
       } catch (IOException ex) {
         logger.warn(ex.toString() + " : Unexpected Exception");
         setClientDisconnectedException(ex);
+      } catch (AuthenticationRequiredException ex) {
+        logger.warn(ex.toString() + " : Unexpected Exception");
```

Comment:
Not sure about what level to set these logs, but this seems acceptable...

Just for discussion, Should this be catch the generic GemFireSecurityException?  (I guess we can worry about that if another issue comes up)

Reply:
this is just for extra logging? Is the previous logging "Failed to bind the subject of uniqueId{}...." not enough?

Reply:
No, it's to prevent the exception from being logged as a stack trace.  Without catching, the exception is logged due to being an unhandled exception